### ISSUE TITLE
Glycogen Storage Disorder Type 9 (GSD IX) / PhK Deficiency

### ISSUE THREAD

**Issue by EngageHealthResearch** [obophenotype/human-phenotype-ontology#7634]
**Preferred term label:**
Glycogen Storage Disorder Type 9 

**Synonyms**
(GSD IX) / PhK Deficiency / Glycogen storage disease due to phosphorylase kinase deficiency 

**Definition (free text, please give PubMed ID)**
https://www.agsdus.org/docs/GSD-VI-and-IX-Practice-Guidelines-2019.pdf 

**Parent term (use hpo.jax.org/app)**
GSD

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:370

**Your nano-attribution (ORCID)**
[GSD-VI-and-IX-Practice-Guidelines-2019.pdf](https://github.com/obophenotype/human-phenotype-ontology/files/9385049/GSD-VI-and-IX-Practice-Guidelines-2019.pdf)

**Comment by pnrobinson** [comment]
@EngageHealthResearch  the HPO is for phenotypic features of disease, but does not include terms for the diseases themselves. For the latter, one of the resources is Mondo. @nicolevasilevsky  -- I think that we need to revise Mondo in this area, we do not seem to have terms for all subforms of Glycogen Storage Disorder Type 9

**Comment by EngageHealthResearch** [comment]
@pnrobinson - thanks for the clarification. I'm brand new to this. Here's a bit more background around my request. I called NCATS/GARD in order to notify them that they did not have GSDIX. The information specialists told me that they pull data from HPO and that I should put in a request via GitHub to have a "table" created. The article I attached does talk about the clinical similarities between GSDVI and GSDIX. Any other advice/comments is welcomed and appreciated.

**Comment by pnrobinson** [comment]
So GSDIX is a group of diseases. The HPO website has information about the individual diseases, 
e.g. IXa https://hpo.jax.org/app/browse/disease/OMIM:306000
IXb https://hpo.jax.org/app/browse/disease/OMIM:261750
IXc https://hpo.jax.org/app/browse/disease/OMIM:613027

Could I ask what your use case? Maybe I can then make a suggestion as to how to move forward.

**Comment by nicolevasilevsky** [comment]
@pnrobinson I think the disease described in the paper is equivalent to IXa (https://omim.org/entry/306000, MONDO:0010598).

This used to be called Glycogen storage disease 8, and this is the (now obsolete) GARD page:  https://rarediseases.info.nih.gov/diseases/6538/glycogen-storage-disease-8

It looks like GARD does not have a page for Glycogen Storage Disease Ixa with our HPO annotations. @EngageHealthResearch I think they best thing for you to do is to ask GARD to create page based on the HPO annotations listed here: https://hpo.jax.org/app/browse/disease/OMIM:306000

I am cc'ing Gioconda from GARD: @galyea-gard

**Comment by galyea-gard** [comment]
Hi Nicole! Thank you for your message. As I commented to you, we are trying to make sure that all the suggestions and concerns regarding to terms (new terms, obsolete terms, etc)  will be addressed as soon as possible by the staff that is in charge for the content of the new GARD website but the process is taking more time than before.

Phosphorylase kinase (PhK) deficiency causing glycogen storage disease type IX (GSD IX) results from deficiency of the enzyme phosphorylase b kinase, which has four copies each of four subunits (α, β, γ, and δ). Pathogenic variants in:

  *   PHKA1, encoding subunit α, cause muscle PhK deficiency (glycogen storage disease IXd (GDS9D), also known as X-linked muscle phosphorylase kinase deficiency) https://omim.org/entry/300559
  *   PHKA2, also encoding subunit α, cause the most common form, liver PhK deficiency - equivalent to glycogen storage disease type IXa1 (GSD9A1) and to type IXa2 (GSD9A2) The one you mentioned: https://omim.org/entry/306000 , MONDO:0010598)
  *   PHKB, encoding subunit β, cause  PhK deficiency in both liver and muscle (glycogen storage disease type IXb (GSD9B)) - See OMIM https://omim.org/entry/261750
  *   PHKG2, encoding subunit γ, cause  liver PhK deficiency (glycogen storage disease IXc (GSD9C)) See OMIM https://omim.org/entry/613027

Notice that Orphanet classifies these disorders according to the involved organ:

"GSD due to liver PhK deficiency is due to mutations in the PHKA2 or PHKG2 genes encoding the liver isoforms of the alpha and gamma subunits of PhK. Transmission is X-linked and autosomal recessive, respectively. GSD due to liver and muscle PhK deficiency is transmitted in an autosomal recessive manner and is due to mutations in the PHKB gene which encodes the beta subunit of PhK. The muscle-specific isoforms of the alpha and gamma subunits of PhK are encoded by the PHKA1 and PHKG1<https://www.omim.org/entry/172470> genes respectively, but until now mutations have only been found in the PHKA1 gene in patients with GSD due to muscle PhK deficiency. Transmission is X-linked". See: https://www.orpha.net/consor/cgi-bin/Disease_Search.php?lng=EN&data_id=357 Orphanet has the following subtypes:


  *   Glycogen storage disease due to liver and muscle phosphorylase kinase deficiency<https://www.orpha.net/consor/cgi-bin/Disease_Search.php?lng=EN&data_id=11266&disease=Glycogen-storage-disease-due-to-liver-and-muscle-phosphorylase-kinase-deficiency&search=Disease_Search_Simple> ORPHA:79240 - This is GSD type IXb - (OMIM: 261750<http://omim.org/entry/261750>)  We do not have this one in GARD.
  *    Glycogen storage disease due to liver phosphorylase kinase deficiency<https://www.orpha.net/consor/cgi-bin/Disease_Search.php?lng=EN&data_id=20103&disease=Glycogen-storage-disease-due-to-liver-phosphorylase-kinase-deficiency&search=Disease_Search_Simple> ORPHA:264580 This is for both GSD type IXa1, IXa2 and also for GSD IXc.  (OMIM 306000<http://omim.org/entry/306000> for IXa1 and IXa2 and 613027<http://omim.org/entry/613027> for GSDIXc) - which, again, is the obsolete mentioned in your message GSD8)
  *    Glycogen storage disease due to muscle phosphorylase kinase deficiency<https://www.orpha.net/consor/cgi-bin/Disease_Search.php?lng=EN&data_id=677&disease=Glycogen-storage-disease-due-to-muscle-phosphorylase-kinase-deficiency&search=Disease_Search_Simple> ORPHA:715 This is for GSD IXc (OMIM https://omim.org/entry/613027 )  and GSD IXd  In GARD we have "Glycogen storage disease due to muscle phosphorylase kinase (PhK) deficiency due to mutations in PHA1 in GARD: https://rarediseases.info.nih.gov/diseases/3858/muscular-phosphorylase-kinase-deficiency


We may follow Orphanet classification.  I do not know how much time it will take to do this but, as commented, I am forwarding your message and my suggestions to the staff in charge of the content.

Thank you! Gioconda

From: Nicole Vasilevsky ***@***.***>
Sent: Tuesday, August 23, 2022 2:51 PM
To: obophenotype/human-phenotype-ontology ***@***.***>
Cc: Alyea, Gioconda (NIH/NCATS) [C] ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [obophenotype/human-phenotype-ontology] Glycogen Storage Disorder Type 9 (GSD IX) / PhK Deficiency (Issue #7634)


@pnrobinson<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fpnrobinson&data=05%7C01%7Cgioconda.alyea%40nih.gov%7C7b470d12995d48df0c9208da8538646d%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C637968774508340349%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=IeeEXibYqjkTHiBgOtPb%2FY6dHqyh0aRP1TCkvZ4R%2FRc%3D&reserved=0> I think the disease described in the paper is equivalent to IXa (https://omim.org/entry/306000<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fomim.org%2Fentry%2F306000&data=05%7C01%7Cgioconda.alyea%40nih.gov%7C7b470d12995d48df0c9208da8538646d%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C637968774508340349%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=ocXDCGcQN7BRTKVh7oCrVGT85BkfDDgs3eQIT7O90LA%3D&reserved=0>, MONDO:0010598).

This used to be called Glycogen storage disease 8, and this is the (now obsolete) GARD page: https://rarediseases.info.nih.gov/diseases/6538/glycogen-storage-disease-8

It looks like GARD does not have a page for with our HPO annotations. @EngageHealthResearch<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FEngageHealthResearch&data=05%7C01%7Cgioconda.alyea%40nih.gov%7C7b470d12995d48df0c9208da8538646d%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C637968774508340349%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=RnhAwssCj9D9j0hRZ9WBeWGxA0lW1iMYhrKsI6eBN1Q%3D&reserved=0> I think they best thing for you to do is to ask GARD to create page based on the HPO annotations listed here: https://hpo.jax.org/app/browse/disease/OMIM:306000<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fhpo.jax.org%2Fapp%2Fbrowse%2Fdisease%2FOMIM%3A306000&data=05%7C01%7Cgioconda.alyea%40nih.gov%7C7b470d12995d48df0c9208da8538646d%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C637968774508340349%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=S5JXxaWrw%2FGYNpkcYToRNza5a717w1DivW0iHqso2dI%3D&reserved=0>

I am cc'ing Gioconda from GARD: @galyea-gard<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fgalyea-gard&data=05%7C01%7Cgioconda.alyea%40nih.gov%7C7b470d12995d48df0c9208da8538646d%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C637968774508340349%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=djZ9%2F8xFYBJsao1DtjBh%2FDyEj%2FMvsNnuLwYEJqELhWs%3D&reserved=0>

-
Reply to this email directly, view it on GitHub<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fobophenotype%2Fhuman-phenotype-ontology%2Fissues%2F7634%23issuecomment-1224609270&data=05%7C01%7Cgioconda.alyea%40nih.gov%7C7b470d12995d48df0c9208da8538646d%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C637968774508340349%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=HYrWfWdMBz1J3aIbTQ4KtYODJDsfQeMqYehnHUdiN0Q%3D&reserved=0>, or unsubscribe<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAPG3MG3B5VLRWKARWREUQFLV2UMQHANCNFSM57B2SDAA&data=05%7C01%7Cgioconda.alyea%40nih.gov%7C7b470d12995d48df0c9208da8538646d%7C14b77578977342d58507251ca2dc2b06%7C0%7C0%7C637968774508340349%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=EejVi0EEloHdwHkBg7DILpMcxV4qjEs1jvoVWC%2FgTsE%3D&reserved=0>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>
CAUTION: This email originated from outside of the organization. Do not click links or open attachments unless you recognize the sender and are confident the content is safe.

**Comment by pnrobinson** [comment]
Hi @EngageHealthResearch  -- this issue is getting pretty complex and I want to make sure we have addressed your concerns before closing it. I think what we want to do is to ensure that Mondo has a representation of all diseases and HPO has a representation of all of the phenotypic features (symptoms) of the diseases. If this is taken care of please close this issue but if there are open questions it would probably be best to zoom -- could you send Peter Robinson AT JAX ORG an email in this case?

**Comment by pnrobinson** [comment]
@EngageHealthResearch  -- I believe HPO has all relevant genetic types of GSDIX, and each of these types has a table of annotations. I will close this issue now but if something is wrong or missing, please send me an email as mentioned above and we can set up a zoom to discuss how to move forward. Thanks!
